cat.md

```diff
@@ -40,9 +46,30 @@ Cat Semantics
 
     syntax CatStep ::= "bite" String Address
  // ----------------------------------------
+    rule <k> Cat . bite ILK URN
+          => call Vat . grab ILK URN THIS VOWADDR (-1 *Rat minRat(INK,LUMP)) (-1 *Rat minRat(ART, minRat(INK,LUMP) *Rat ART /Rat INK))
+          ~> call Vow . fess (ART *Rat RATE)
+          ~> call Flip FLIP . kick URN VOWADDR (ART *Rat RATE *Rat CHOP) minRat(INK,LUMP) 0
+         ...
+         </k>
+         <this> THIS </this>
+         <cat-live> true </cat-live>
+         <cat-ilks>...
+           ID |-> Ilk(... flip: FLIP, chop: CHOP, lump: LUMP)
```

Comment:
Should this be `ILK |-> ...`? `ID` isn't bound anywhere, so you're grabbing a random one.